source/includes/api/_layer.md

```diff
@@ -807,3 +807,291 @@ userName                | String         | No (autogenerated)   | null
+
+#### Errors for deleting a layer
+
+Error code     | Error message           | Description
+-------------- | ----------------------- | ---------------------------------------------------------------------------------
+404            | Layer not found         | No contextual layer was found for the provided id.
+500            | Layer retrieval failed. | An error occurred while fetching the layer information from the database.
+500            | Team retrieval Failed.  | An error occurred while fetching the information about the team with id provided.
+500            | Layer update failed.    | An error occurred while saving the updated information of the contextual layer.
+
+### Getting loss layer tiles
+
+> Example request to fetch loss layer tiles:
+
+```shell
+curl -X GET "https://api.resourcewatch.org/v1/contextual-layer/loss-layer/:startYear/:endYear/:z/:x/:y.png"
+```
+
+Use this endpoint to retrieve PNG image corresponding to the loss layer tiles for the `(x,y,z)` tuple (where `x` represents latitude, `y` longitude, and `z` the zoom level) for loss layers for the interval of years from `startYear` to `endYear`.
+
+If successful, this endpoint returns a PNG file of the loss layer tile.
+
+#### Errors for deleting a layer
+
+Error code     | Error message           | Description
+-------------- | ----------------------- | ---------------------------------------------------------------------------------
+404            | Tile not found          | No tile was found for the parameters provided.
+
+### Contextual layer reference
+
+This section gives you a complete view of the properties that are maintained as part of a contextual layer. When interacting with a contextual layer (on get, on create, etc) you will find most of these properties available to you, although they may be organized in a slightly different structure (ie: on get, everything but the `id` is nested inside an `attributes` object).
+
+You can find more details in the [source code](https://github.com/gfw-api/fw-contextual-layers/blob/dev/app/src/models/layer.model.js).
+
+Field name  | Type    | Required            | Default value  | Description                                                                  
+----------- | ------- | ------------------- |--------------- | -------------------------------------- 
+id          | String  | Yes (autogenerated) |                | Unique Id of the contextual layer. Auto-generated on creation. Cannot be modified by users.
+name        | String  | Yes                 |                | Name of the contextual layer.              
+description | String  | No                  |                | Description of the contextual layer.              
+url         | String  | Yes                 |                | URL to fetch tiles for the contextual layer.              
+isPublic    | Boolean | Yes                 | false          | If the contextual layer is publicly accessible.
```

Comment:
I thought just `name` and `url` were required based on the language above?

Reply:
As defined in the Mongoose model, `isPublic` is required. But it has a default value, so it's not required :sweat_smile: I understand it makes the docs sound a bit confusing, so I'll update the table marking `isPublic` as not required.

Reply:
(I'll update the `enabled`, `owner.type` and `owner.id` attributes for the same reasons, to avoid confusion)